### PR TITLE
View on Portal States (BS Alerts and Button)

### PR DESCRIPTION
**Pull request by JVickery-TBS**
image (1)
image

### CODE REVIEW

**Review by wardi — APPROVED** [review]

**Comment by JVickery-TBS** [comment]
Just waiting for translations for the new strings

**Comment by wardi** [comment]
looks good

**Comment by JVickery-TBS** [comment]
Added another new alert for "Record missing approval from the Information Management Senior Official (IMSO)"

And updated the `imso_approval` field label and description to include IMSO. Will most likely be changing the test once it is finalized. And then adding in the translations later.

File: ckanext/canada/templates/internal/package/read.html
Comment (PatLittle) [inline ckanext/canada/templates/internal/package/read.html:1]: I think the button&messages are missing the /dataset/ path

Comment (JVickery-TBS) [reply ckanext/canada/templates/internal/package/read.html:1]: fixed

**Pull request merged**